modules/op-results-opc-pipelinerunresults.adoc

```diff
@@ -0,0 +1,114 @@
+----
++
+.Example output of the `opc results pipelinerun describe` command
+[source]
+----
+Name: operator-main-index-4-18-on-pull-request-7kssl
+Namespace: tekton-ecosystem-tenant
+Service Account: appstudio-pipeline
+Labels:
+ app.kubernetes.io/managed-by=pipelinesascode.tekton.dev
+ app.kubernetes.io/version=v0.33.0
+Annotations:
+ appstudio.openshift.io/snapshot=openshift-pipelines-main-b7jj6
+ build.appstudio.openshift.io/repo=https://github.com/openshift-pipelines/operator?rev=ba5e62e51af0c88bc6c3fd4201e789bdfc093daa
+
+📌 Status
+STARTED          DURATION         STATUS
+27d ago          9m54s            Succeeded
+
+⏱ Timeouts
+Pipeline:   2h0m0s
+
+⚓ Params
+  NAME                          VALUE
+  • git-url                     https://github.com/pramodbindal/operator
+  • revision                    ba5e62e51af0c88bc6c3fd4201e789bdfc093daa
+
+🗂  Workspaces
+  NAME                SUB PATH            WORKSPACE BINDING
+  • workspace          ---                VolumeClaimTemplate
+  • git-auth           ---                Secret (secret=pac-gitauth-ceqzjt)
+
+📦 Taskruns
+  NAME                                                                         TASK NAME
+  • operator-main-index-4-18-on-pull-request-7kssl-init                        init
+  • operator-main-index-4-18-on-pull-request-7kssl-clone-repository            clone-repository
+----
+
+* To view the full YAML manifest of the pipeline run, enter the following command:
++
+[source,terminal]
+----
+$ opc results pipelinerun describe -n <namespace_name> --output yaml <pipelinerun_name>
+----
++
+Alternatively, you can use the pipeline run UUID instead of the name:
++
+[source,terminal]
+----
+$ opc results pipelinerun describe -n <namespace_name> --output yaml --uid <pipelinerun_uuid>
+----
+
+* To view the logs associated with the pipeline run, enter the following command:
++
+[source,terminal]
+----
+$ opc results pipelinerun logs -n <namespace_name> <pipelinerun_name>
+----
++
+Alternatively, you can use the pipeline run UUID instead of the name:
++
+[source,terminal]
+----
+$ opc results pipelinerun logs -n <namespace_name> --uid <pipelinerun_uuid>
+----
+
+[IMPORTANT]
+====
+Logs that the `opc results pipelinerun logs` displays do not include logs of task runs that completed within this pipeline runs. To view these logs, find the names of the task runs in this pipeline run using the `opc results taskrun list --pipelinerun` command and specify the name of the pipeline run. Then use the `opc results taskrun log` command to view the logs for the task runs.
+====
+
+[NOTE]
+====
+If you do not specify the `-n` option, the `opc` command applies to the current namespace, To set the current namespace, use the `oc project` command.
+====
+
+[NOTE]
+====
+You can use the short form `opc results pr desc` instead of `opc results pipelinerun describe`.
+
+You can use the short form `opc results pr logs` instead of `opc results pipelinerun logs`.
+====
```

Comment:
How much of this needs to be here vs. either in the tool's guidance or organically in documented task flows? I'd try to avoid stacking admonitions unless necessary.

Reply:
The flows get tremendously overfilled if this is squeezed in. Faced with this issue I added a reference module for short names and just dropped the information about not specifying the namespace.